src/Pos/Generator/Block/Mode.hs

```diff
@@ -0,0 +1,245 @@
+-- | Execution mode used by blockchain generator.
+
+module Pos.Generator.Block.Mode
+       ( MonadBlockGenBase
+       , MonadBlockGen
+       , BlockGenContext (..)
+       , BlockGenMode
+       , mkBlockGenContext
+
+       , InitBlockGenContext (..) -- useless
```

Comment:
You did it? 🙄 

I think it's fine to just export it as a workaround. I didn't want to think about smart solution.